documentation/cluster-controller/cluster-controller.adoc

```diff
@@ -0,0 +1,267 @@
+= Cluster Controller
+
+The cluster controller is in charge to deploy a Kafka cluster alongside a Zookeeper ensemble. It's also able to deploy a 
+Kafka Connect cluster which connects to an already existing Kafka cluster.
+
+When up and running, it starts to "watch" for ConfigMaps containing the Kafka or Kafka Connect cluster configuration.
+Such a ConfigMaps have to have a specific label which by default is `strimzi.io/kind=cluster` (as described later) that 
+can be changed through a corresponding environment variable.
+
+When a new ConfigMap is created on the Kubernetes/OpenShift cluster, the controller gets the deployment configuration from
+its `data` section and starts the creation process of a new cluster (Kafka or Kafka Connect). It's done creating a bunch 
+of Kubernetes/OpenShift resources needed for such a deployment, like StatefulSets, ConfigMaps, Services and so on.
+
+Every time the ConfigMap is updated by the user with some changes in the `data` section, the controller executes a related updating
```

Comment:
_executes a related updating operation_ sounds strange. I also don't think we need to get into the details about the operations etc. Lets just say _the cluster controller updates the cluster_.

Reply:
I think that it's better for the user knowing what's going on from a Kubernetes/OpenShift perspective.